POUP2/scenes/player/scripts/player.gd

```diff
@@ -1,58 +1,59 @@
-extends Area2D
+extends CharacterBody2D
 signal hit
 signal enemy_kill
 
-@export var speed = 400 # How fast the player will move (pixels/sec).
+@export var speed := 400 # How fast the player will move (pixels/sec).
 @export var attack_scene: PackedScene
-var screen_size # Size of the game window.
+var fixed := true
```

Comment:
**question:** What does this syntax mean?

Reply:
its a shorthand to assign a variable with a fixed type infered from the value you assign to it
(allows to have strictly static typed variable without specifying the type explicitely)

Reply:
_Stylé_ ™️